examples/cpp_sycl/source/kmeans/kmeans_distr_dense_batch.cpp

```diff
@@ -0,0 +1,90 @@
+/* file: kmeans_dense_batch.cpp */
```

Comment:
What is purpose of that example? `kmeans_distr::stepless` is the same as kmeans batch example.

Reply:
Removed the example and bat files too